zaza/utilities/openstack.py

```diff
@@ -1123,6 +1124,54 @@ def get_application_config_keys(application):
     return list(application_config.keys())
 
 
+def get_current_os_release_pair(application='keystone'):
+    """Return OpenStack Release pair name.
+
+    :param application: Name of application
+    :type application: string
+    :returns: Name of the OpenStack release pair
+    :rtype: str
+    :raises: exceptions.NoKeystoneFound
+    :raises: exceptions.SeriesNotFound
+    :raises: exceptions.OSVersionNotFound
+    """
+    keystone = juju_utils.get_machines_for_application(application)
```

Comment:
Can you change the name of this variable from 'keystone' please?

Reply:
Done